drivers/spi/spi_smartbond.c

```diff
@@ -273,6 +285,32 @@ static int spi_smartbond_init(const struct device *dev)
 	return 0;
 }
 
+#ifdef CONFIG_PM_DEVICE
+static int spi_smartbond_pm_action(const struct device *dev,
+				   enum pm_device_action action)
+{
+	const struct spi_smartbond_cfg *config = dev->config;
+	struct spi_smartbond_data *data = dev->data;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		da1469x_pd_acquire(MCU_PD_DOMAIN_COM);
+		CRG_COM->SET_CLK_COM_REG = config->periph_clock_config;
+		config->regs->SPI_CTRL_REG = data->spi_ctrl_reg;
+		break;
+	case PM_DEVICE_ACTION_SUSPEND:
+		config->regs->SPI_CTRL_REG &= ~SPI_SPI_CTRL_REG_SPI_ON_Msk;
```

Comment:
Shouldn't we also gate the SPI clock by setting `CRG_COM->RESET_CLK_COM_REG = config->periph_clock_config`;

Reply:
I did not include this only because power saving was only mentioned in SPI_CTR_REG.SPI_EN bit and when measuring current I did not noticed any difference.
In may case when device went to sleep (all PD_COM was down) register was cleared anyway.
If you think it can reduce power consumption when device is put to sleep and SOC is not, I don't have any problem adding this clock dismissal.